fml/platform/fuchsia/message_loop_fuchsia.h

```diff
@@ -25,6 +27,10 @@ class MessageLoopFuchsia : public MessageLoopImpl {
   void WakeUp(fml::TimePoint time_point) override;
 
   zx_handle_t timer_;
+
+  async::Wait* timer_wait_;
```

Comment:
`std::unique_ptr<async::Wait>` or a stack object instead of raw pointers please.